### PR TITLE
added new ramls for data import queue

### PR DESCRIPTION
**Pull request by evaluk**
New ramls for related change in https://github.com/folio-org/mod-data-import/pull/258

### CODE REVIEW

**Review by ncovercash — APPROVED** [review]
LGTM

File: schemas/mod-data-import/dataImportQueueItem.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "description": "job execution Id",
+        "$ref": "../../raml-util/schemas/uuid.schema"
+      },
+      "uploadDefinitionId": {
+        "description": "upload Definition Id",
+        "$ref": "../../raml-util/schemas/uuid.schema"
+      },
+      "size": {
+        "description": "Size of the file",
+        "type": "integer"
```
Comment (carolegodfrey) [inline schemas/mod-data-import/dataImportQueueItem.json:21]: Is this the size of the file (in bytes or records)?

File: schemas/mod-data-import/dataImportQueueItem.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      "originalSize": {
+        "description": "original size of the file",
+        "type": "integer"
```
Comment (carolegodfrey) [inline schemas/mod-data-import/dataImportQueueItem.json:25]: Is this the size of the parent file (if the original file was split)?
Is it the size in bytes or records?

File: schemas/mod-data-import/dataImportQueueItem.json (hunk 1 above)
Comment (evaluk) [reply schemas/mod-data-import/dataImportQueueItem.json:21]: I dont have a preference here.  We should talk about it but I am going to merge this PR

File: schemas/mod-data-import/dataImportQueueItem.json (hunk 2 above)
Comment (evaluk) [reply schemas/mod-data-import/dataImportQueueItem.json:25]: I clarified a bit on this comment.

**Review by ncovercash — APPROVED** [review]

**Pull request merged**